src/Components/Facility/PatientNotesList.tsx

```diff
@@ -11,6 +14,7 @@ import { NoteType } from "./PatientNoteCard";
 interface PatientNotesProps {
   patientId: any;
   facilityId: any;
+  consultationId?: any;
```

Comment:
you could use `useSlug` / `useSlugs` to avoid passing path params as props to deeply nested components.